Capfile

```diff
@@ -13,8 +13,6 @@ require 'capistrano/maintenance'
 
 require 'dlss/docker/capistrano'
 require 'capistrano/bundler'
-require 'dlss/capistrano'
-require 'capistrano/honeybadger'
```

Comment:
The documentation of https://github.com/sul-dlss/dlss-capistrano-docker suggests we don't need these here